### PR TITLE
Clarify RebootMethod

### PR DESCRIPTION
**Pull request by dplore**

### CODE REVIEW

File: system/system.proto [hunk 1]
```diff
@@ -118,6 +118,8 @@ message RebootResponse {
 // A RebootMethod determines what should be done with a target when a Reboot is
 // requested.  Only the COLD method is required to be supported by all
 // targets.  Methods the target does not support should result in failure.
+// 
+// If a target does not support a method, `INVALID_ARGUEMENT` should be returned.
```
Comment (greg-dennis) [inline system/system.proto]: ARGUEMENT->ARGUMENT

Comment (greg-dennis) [inline system/system.proto]: Is this line redundant with the last line of the preceding paragraph?

File: system/system.proto [hunk 2]
```diff
@@ -118,6 +118,9 @@ message RebootResponse {
 // A RebootMethod determines what should be done with a target when a Reboot is
 // requested.  Only the COLD method is required to be supported by all
 // targets.  Methods the target does not support should result in failure.
+// 
+// If a target does not support a method, `INVALID_ARGUEMENT` should be
```
Comment (greg-dennis) [inline system/system.proto:122]: Should i also say that INVALID_ARGUMENT should be returned if the default/unknown method is provided or is that implicit?

**Review by greg-dennis — CHANGES_REQUESTED** [review]

File: system/system.proto (hunk 1 above)
Comment (dplore) [reply system/system.proto]: fixed

Comment (dplore) [reply system/system.proto]: fixed

File: system/system.proto (hunk 2 above)
Comment (dplore) [reply system/system.proto]: Added explicity reference for both scenarios.

**Review by greg-dennis — APPROVED** [review]

**Comment by dplore** [comment]
I'll wait a bit for any other comments and merge later today

**Review by ejbrever — APPROVED** [review]
LGTM

**Pull request merged**